homeassistant/components/sensor/tautulli.py

```diff
@@ -0,0 +1,133 @@
+"""
+A platform which allows you to get information from Tautulli.
+
+For more details about this component, please refer to the documentation at
+https://www.home-assistant.io/components/sensor.tautulli
+"""
+
+import logging
+import voluptuous as vol
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import (CONF_API_KEY,
+                                 CONF_HOST,
+                                 CONF_MONITORED_VARIABLES,
+                                 CONF_PORT,
+                                 CONF_SSL,
+                                 STATE_UNAVAILABLE)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import (PLATFORM_SCHEMA)
+
+__version__ = '1.1.0'
+
+REQUIREMENTS = ['pytautulli==0.1.4']
```

Comment:
Where is the source code? https://gitlab.com/ludeeus/pytautulli gives me a 404.

Reply:
Yeah... sorry about that I moved it to GitHub and forgot to update the link.
<https://github.com/ludeeus/pytautulli>